README.md

```diff
@@ -1,70 +1,51 @@
-# Getting Started with Create React App
+# Math Magician App 
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> A web app developed using `React`. It's a simple calcualtor application
 
-## Available Scripts
 
-In the project directory, you can run:
+## Built With
 
-### `npm start`
+- HTML5
+- CSS3
+- React JS
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Live Demo (if available)
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+[Live Demo Link](https://livedemo.com)
 
-### `npm test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Getting Started
 
-### `npm run build`
+**`git clone https://github.com/RitobrotoMukherjee/math-magician.git`**
+**`cd math-magician`**
+**`npm run start`**
```

Comment:
- [ ] [Optional] Great job including instructions for starting this project. However, if a person clones this project and changes the directory, the person would need to run `npm install` before running `npm run` start. Kindly include this command. 👍